m4_sessions/1811_inning/timprov.md

```diff
@@ -26,6 +34,11 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Intro
+Whole group: Pass the clap; One word story;
```

Comment:
Can you link these activities so future groups might be able to do this as well?